bip/t3dn_bip/utils.py

```diff
@@ -2,11 +2,15 @@
 
 import bpy
 import sys
-from subprocess import call
 from typing import Tuple
 from zlib import decompress
 from array import array
 
+if sys.platform == 'win32':
+    from .windows import call
+else:
+    from subprocess import call
```

Comment:
For Unix we should run the command via sudo. But we would need to run it in a GUI terminal, so the user can actually see the input.

Reply:
On Linux we would probably have to detect xterm, gnome-terminal and konsole and run it the one, which is available. We have similar detection code for the X11 openers in the browser module.

Reply:
On macOS we could use `open -b com.apple.terminal script.sh`, whereas script.sh is a temporary executable bash script we would generate on the fly.